securedrop/journalist_app/utils.py

```diff
@@ -437,11 +437,20 @@ def set_diceware_password(user: Journalist, password: Optional[str]) -> bool:
         return False
 
     # using Markup so the HTML isn't escaped
-    flash(Markup("<p>" + gettext(
-        "Password updated. Don't forget to "
-        "save it in your KeePassX database. New password:") +
-        ' <span><code>{}</code></span></p>'.format(password)),
-        'success')
+    flash(
+        Markup(
+            "<p>{message} <span><code>{password}</code></span></p>".format(
+                message=Markup.escape(
+                    gettext(
+                        "Password updated. Don't forget to save it in your KeePassX database. "
+                        "New password:"
+                    )
+                ),
+                password=Markup.escape("" if password is None else password)
```

Comment:
In the signature of `set_diceware_password`, the `password` argument is typed as `Optional[str]`, because it's populated with `request.form.get`, if I'm reading it correctly. Using just `Markup.escape(password)` here, `mypy` complains:
```
securedrop/journalist_app/utils.py:449: error: Argument 1 to "escape" of "Markup" has incompatible type "Optional[str]"; expected "str"
```